packages/algjulia-service/src/decapodes-service/initial_conditions.jl

```diff
@@ -42,10 +43,10 @@ GaussianIC(r::Rectangle) = GaussianIC(r, GaussianData(r))
 TaylorVortexIC(d::Sphere) = TaylorVortexIC(d, TaylorVortexData())
 
 
-function initial_conditions(jo::JSON3.Object, uuid2symb::Dict{String, Symbol}, mesh_builder::String)
-    ic_specs = jo[:initialConditions];
+function initial_conditions(json_object::JSON3.Object, geometry::Geometry, uuid2symb::Dict{String, Symbol})
```

Comment:
Should dispatch on `AbstractDict` instead of `JSON3.Object` to avoid unnecessary coupling to JSON3.jl.